deployment/aerie_db_migration.py

```diff
@@ -190,8 +190,8 @@ def reload_metadata(self):
     """
     Apply and reload the metadata.
     """
-    self.execute('metadata apply')
-    self.execute('metadata reload')
+    self.execute(f'metadata apply {self.metadata_suffix} {self.command_suffix}')
+    self.execute(f'metadata reload {self.metadata_suffix} {self.command_suffix}')
```

Comment:
Two things:
1) `command_suffix` is already added in `self.execute`
2) `self.execute` has a parameter for passing a string of flags

Correct syntax (based on how I designed the code) is:
```suggestion
    self.execute('metadata apply', self.metadata_suffix)
    self.execute('metadata reload', self.metadata_suffix)
```